src/transformers/models/layoutlmv2/detectron2_config.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+def add_layoutlmv2_config(cfg):
```

Comment:
Can we add a comment on what is the use of this file and this function?

Reply:
+1 that would be useful